### PR TITLE
feat: Add CVV/AVS and raw response codes

### PR DESCRIPTION
**Pull request by cbetta**
Adds standardized AVS/CVV response codes to the Transaction and TransactionSummary schemas. Also adds raw response codes and descriptions to both schemas.

**Note:** Due to our SDKs lack of support for inheritance we (for now) need to define this twice. This is a known issue.

### CODE REVIEW

File: reference/resources/transaction.yaml [hunk 1]
```diff
@@ -177,3 +177,69 @@ properties:
...
+
+      - `no_match` - neither address or postal code match
+      - `match` - both address and postal code match
+      - `partial_match_address` - address matches but postal code does not
+      - `partial_match_postcode` - postal code matches but address does not
+      - `unavailable ` - AVS is unavailable for card/country
+
+      The value of this field can be `null` if the payment service did not
+      provide a response.
+    example: match
```
Comment (hasier) [inline reference/resources/transaction.yaml:216]: [minor] Should we set this to `partial_match_address` to align it to the rest of the field examples?

File: reference/resources/transaction_summary.yaml [hunk 2]
```diff
@@ -122,3 +122,69 @@ properties:
...
+
+  cvv_response_code:
+    type: string
+    description: |-
+      The response code received from the payment service for the Card
+      Verification Value (CVV). This code is mapped to a standardized Gr4vy
+      CVV response code.
+
+      - `no_match` - the CVV does not match the expected value
+      - `match` -the CVV matches the expected value
```
Comment (hasier) [inline reference/resources/transaction_summary.yaml:178]: [minor] Missing whitespace.

File: reference/resources/transaction_summary.yaml [hunk 3]
```diff
@@ -122,3 +122,69 @@ properties:
...
+
+      - `no_match` - neither address or postal code match
+      - `match` - both address and postal code match
+      - `partial_match_address` - address matches but postal code does not
+      - `partial_match_postcode` - postal code matches but address does not
+      - `unavailable ` - AVS is unavailable for card/country
+
+      The value of this field can be `null` if the payment service did not
+      provide a response.
+    example: match
```
Comment (hasier) [inline reference/resources/transaction_summary.yaml:161]: [minor] Same here about the example.

File: reference/resources/transaction.yaml [hunk 4]
```diff
@@ -177,3 +177,69 @@ properties:
     example: "123456789012345"
     nullable: true
     default: null
+
+  raw_response_code:
+    type: string
+    description: |-
+      This is the authorization code received from the payment service. This
```
Comment (steve-gr4vy) [inline reference/resources/transaction.yaml:184]: 'authorization code' I would say 'This is the response code received..'

File: reference/resources/transaction_summary.yaml [hunk 5]
```diff
@@ -122,3 +122,69 @@ properties:
       - $ref: "../openapi.yaml#/components/schemas/PaymentService--Snapshot"
       - description: |-
           The payment service used for this transaction.
+
+  raw_response_code:
+    type: string
+    description: |-
+      This is the authorization code received from the payment service. This
```
Comment (steve-gr4vy) [inline reference/resources/transaction_summary.yaml:129]: 'authorization code' -> 'response code'

File: reference/resources/transaction.yaml (hunk 1 above)
Comment (cbetta) [reply reference/resources/transaction.yaml:216]: @hasier sorry not sure what you mean with "align to rest of the field examples".

Comment (cbetta) [reply reference/resources/transaction.yaml:216]: Ah I get it

File: reference/resources/transaction_summary.yaml (hunk 2 above)
Comment (hasier) [reply reference/resources/transaction_summary.yaml:178]: Still missing this one 😬

**Review by hasier — DISMISSED** [review]

**Review by hasier — DISMISSED** [review]

**Review by federicojasson — APPROVED** [review]

**Review by steve-gr4vy — APPROVED** [review]

**Pull request merged**